admiral/pkg/controller/common/config.go

```diff
@@ -80,6 +80,10 @@ func GetGlobalTrafficDeploymentLabel() string {
 	return admiralParams.LabelSet.GlobalTrafficDeploymentLabel
 }
 
+func GetRoutingPolicyLabel() string {
+	return admiralParams.LabelSet.WorkloadIdentityKey
```

Comment:
Will be useful to check for nil for each object to prevent panics.

Reply:
Will Add this change